### PR TITLE
Add my card information

### PR DESCRIPTION
**Pull request by leeyoonj**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -103,9 +103,42 @@ <h2>A project for first-time contributions</h2>
         <!-- ________*END TEMPLATE* Contributor card END ________  -->
 
         <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
+
         <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (dilshankarunarathne) [inline index.html:111]: Please change this comment 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` 
into `<!-- ________ Your_Name Contributor card START ________ -->`

File: index.html [hunk 2]
```diff
@@ -103,9 +103,42 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://javascript.info/" target="_blank" title="JavaScript.Info">
+                  JavaScript.Info
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (dilshankarunarathne) [inline index.html:140]: Please change this comment as well. 
`<!-- ________*END TEMPLATE* Contributor card END ________ -->` 
to `<!-- ________ Your_Name Contributor card END ________ -->`

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
Hello @leeyoonj, thank you for contributing to this project. ❤
But there are a couple of small changes that need to be done before we go ahead and merge your code. Please change these two comments, because they are very helpful for everyone to identify each card. 
Once you've made those changes, please get back to us. We will be able to review and merge your pull request in a flash.

**Comment by leeyoonj** [comment]
changes made

**Comment by Preet-Sojitra** [comment]
Congratulationss @leeyoonj Your pull request has been merged. Thanks for contribution :)

**Pull request merged**